support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -261,15 +261,7 @@ const getSubscriptionCopy = (
 			// paperAndDigital(countryGroupId, state.common.referrerAcquisitionData, state.common.abParticipations),
 			premiumApp(countryGroupId),
 		];
-	} else if (
-		[
-			EURCountries,
-			AUDCountries,
-			NZDCountries,
-			GBPCountries,
-			International,
-		].includes(countryGroupId)
-	) {
+	} else if (countryGroupId === EURCountries) {
```

Comment:
this changes the /subscribe page ordering back to how it was before the Aus + France offers, with digisub at the top except in Europe